.github/workflows/esmeta-yetcheck.yml

```diff
@@ -0,0 +1,77 @@
+name: 'esmeta yet phrases detection'
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  esmeta-new-phrases:
+    runs-on: ubuntu-latest
+
+    env:
+      ESMETA_HOME: vendor/esmeta
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
+      - name: Setup SBT
+        uses: sbt/setup-sbt@v1
+      - name: download esmeta
+        run: |
+          mkdir -p "${ESMETA_HOME}"
+          cd "${ESMETA_HOME}"
+          git init
+          git remote add origin https://github.com/es-meta/esmeta.git
+          git fetch --depth 1 origin bd58590ef6badabbe71afdefa02dc32274902621 ;# v0.7.1
```

Comment:
🤔 it's a little unfortunate that this could allow esmeta to be out of sync in the different CI actions. I think we should keep this in a separate file that's read by both CI scripts.

Reply:
In the commit https://github.com/tc39/ecma262/pull/3721/commits/12908abb4792dffa87f843c22dd47c0f2bea299b, I created a unified action `esmeta-installer` to use it for both `esmeta-tycheck` and `esmeta-yetcheck`. From now on, it is enough to modify a single `esmeta-installer/action.yml` file to update the version of ESMeta.